### PR TITLE
Update WcfServerCheck.cs

### PR DESCRIPTION
**Pull request by samsp-msft**
**PR Description**
Changes text when encountering an SVC file.
- [x] AKA.ms link needs to be updated once https://github.com/microsoft/dotnet-blog/pull/815 has been posted.

Addresses #1110

### CODE REVIEW

**Comment by brandonh-msft** [comment]
I'm curious about the move of the links - it looks like the link we have today directs folks on how to change from WCF -> gRPC, but the one you're proposing is instructional for WCF -> CoreWCF. Is that correct?
The message we give them, then, says that _both_ are actually suitable... Is there a way we could give them **both** links so they can more easily figure out which suits their application? Or is there a reason why we're seemingly switching to prefer CoreWCF over gRPC when we say both are suitable replacements?

**Comment by sunandabalu** [comment]
> I'm curious about the move of the links - it looks like the link we have today directs folks on how to change from WCF -> gRPC, but the one you're proposing is instructional for WCF -> CoreWCF. Is that correct? The message we give them, then, says that _both_ are actually suitable... Is there a way we could give them **both** links so they can more easily figure out which suits their application? Or is there a reason why we're seemingly switching to prefer CoreWCF over gRPC when we say both are suitable replacements?

Agreed, since we are mentioning both CoreWCF and gRPC as viable options, providing both the links make sense.

**Comment by samsp-msft** [comment]
Updated to show both links.

**Review by brandonh-msft — APPROVED** [review]

File: src/components/Microsoft.DotNet.UpgradeAssistant/Checks/WcfServerCheck.cs [hunk 1]
```diff
@@ -16,9 +16,10 @@ namespace Microsoft.DotNet.UpgradeAssistant.Checks
     /// </summary>
     public class WcfServerCheck : IUpgradeReadyCheck
     {
-        private const string UPGRADE_LINK = "https://aka.ms/migrate-wcf-to-grpc";
+        private const string WCF_LINK = "https://aka.ms/CoreWCF/migration";
+        private const string gRPC_LINK = "https://aka.ms/migrate-wcf-to-grpc";
```
Comment (brandonh-msft) [inline src/components/Microsoft.DotNet.UpgradeAssistant/Checks/WcfServerCheck.cs:20]: Build's failing cuz of this
```suggestion
        private const string GRPC_LINK = "https://aka.ms/migrate-wcf-to-grpc";
```

**Review by sunandabalu — APPROVED** [review]
LGTM, you might need to change the variable name to be all caps to avoid build breaks.

**Pull request merged**